tests/test_premium_targets.py

```
@@ -201,7 +202,7 @@ def test_premium_targets_get_summary(self):
 
     @pytest.mark.parametrize("sex", ["female", "male"])
     @pytest.mark.parametrize("lang", ["ru", "en", "es"])
-    def test_premium_targets_invariance(self, sex: str, lang: str):
+    def test_premium_targets_invariance(self, sex: Literal["female", "male"], lang: str):
```

Ruff (0.13.1)

Unused method argument: `lang`

(ARG002)